MapboxNavigation/VisualInstructionComponent.swift

```diff
@@ -5,16 +5,20 @@ extension VisualInstructionComponent {
     
     static let scale = UIScreen.main.scale
     
-    func cacheKey() -> String? {
+    var cacheKey: String? {
         switch type {
         case .exit, .exitCode:
             guard let exitCode = self.text else { return nil}
             return "exit-" + exitCode + "-\(VisualInstructionComponent.scale)-\(hashValue)"
-        case .image, .text:
-            guard let imageURL = imageURL else { return nil }
+        case .image:
+            guard let imageURL = imageURL else { return genericCacheKey }
             return "\(imageURL.absoluteString)-\(VisualInstructionComponent.scale)"
-        case .delimiter:
+        default:
```

Comment:
It might make more sense to keep `.delimiter` here and remove default so as new cases are added, the compiler will yell at us to handle them.